config_quick_debug.yml

```diff
@@ -7,8 +7,14 @@ scenario_params_list:
    aggregation_weighting: 'data_volume'
    single_partner_test_mode: 'global'
    minibatch_count: 5
-   is_quick_demo: True
    methods:
     - "Shapley values"
-    - "Independant scores"
+    - "Independant scores" 
     - "TMCS"
+    # - "ITMCS"
```

Comment:
You should put all these methods in here https://github.com/SubstraFoundation/distributed-learning-contributivity/blob/167ad986027712cf98d879ec913bb9950c0b2826/scenario.py#L94
Otherwise they won't run.

Reply:
Oh yeah, i had already put them here but I removed it but error at my last commit

Reply:
I 've corrected it